applications/FluidTransportApplication/python_scripts/apply_scalar_constraint_function_process.py

```diff
@@ -1,5 +1,6 @@
 import KratosMultiphysics

```

Comment:
It seems to me that you are changing this file for each problem. I think that you'd make your life easier by using the `assign_scalar_variable_process` from the core, which does essentially what you are doing in here but with enhanced capabilities.

If you want to avoid it for the moment, you can overwrite the `AppliBoundaryConditions` method in your custom analysis stage before running the problem.

Reply:
I am doing this just for an ad hoc case in which I apply the conditions just to particular nodes. Thank you anyway.